src/main/java/com/mentalhealthapi/mentalhealthapi/MentalHealthController.java

```diff
@@ -29,15 +29,19 @@ public String index() {
      */
     @GetMapping("/disorders")
     public ResponseEntity fetchAllDisorders() {
-        return new ResponseEntity(HttpStatus.OK);
+        try {
+            return new ResponseEntity(HttpStatus.OK);
+        } catch (Exception e) {
+            return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
+        }
     }
```

Comment:
Error handling and proper feedback to the end user can be extremely crucial. It can be done using a try catch like so, or you can create an @exceptionhandler or @ControllerAdvice.

Reply:
Error handling is a good idea, and returning 500 from an exception is definitely the right thing to do.

I'd fill this out a bit more, though.  As it is, return new ResponseEntity(HttpStatus.OK) will never throw an exception, so this catch will never be invoked.

Reply:
Yes, this is a great suggestion. We will handle these errors in our later iterations when there is something to throw an error about. 

Thank you